.github/workflows/ci.yml

```diff
@@ -55,10 +55,12 @@ jobs:
         run: |
           cd ./kibana
           yarn kbn bootstrap --oss
-      - name: Run lint
+      - name: Run check
         run: |
           cd ./kibana/plugins/opendistro_security
           yarn lint
+          yarn tsc
```

Comment:
you may want to move `Run check` after `Hack step to update kibana source to unblock tsc` here, otherwise, tsc will fail due to an unused annotation in Kibana source code

Reply:
That worked. Thanks!